telegram/_message.py

```diff
@@ -3522,6 +3526,19 @@ def _parse_markdown(
                             "Spoiler entities are not supported for Markdown version 1"
                         )
                     insert = f"||{escaped_text}||"
+                elif entity.type == MessageEntity.CUSTOM_EMOJI:
+                    if version == 1:
+                        # this ensures compatibility to previous PTB versions
+                        insert = escaped_text
```

Comment:
I don't quite understand the comment. Do you mean that before the introduction of this entity type, the alternative emoji would was always being used? By that logic we shouldn't raise ValueError for strikethrough etiher, should we? My point is: I had expected to raise an exception here, too :D

Reply:
Well we didn't before, because previously it just wasn't supported, so we used the replacement emoji instead. And I would stick to that, it makes more sense to me then raising an error.

Reply:
My quesiton is: If instead of including a custom emoji markup, we fall back to the static emoji, shouldn't we then also fallback to the ordinary text for strikethrough text? i.e. just leave out the markup? I feel like raising for strikethrough but falling back to the static emoji is an inconsistency …

Reply:
It is inconsistent, yes. But unlike strikethrough, where we introduced this ValueError when it got added to the MessageEntities right away, we decided to fallback to static emojis when custom emojis got added. No matter the markdown flavor. So I think it would be weird to break this just to keep it consistent.

Maybe we do our fancy new deprecation move here?

Reply:
Fair point, makes sense to me, then. But yes, let's deprecate that for md v1 and raise ValueError on the next api update 👍